README.md

```diff
@@ -204,6 +210,47 @@ public class MainApplication extends Application implements ReactApplication {
 }
 ```
 
+###### **Kotlin:**
+
+```kotlin
+// MainApplication.kt
+mport com.adobe.marketing.mobile.AdobeCallback;
```

Comment:
`import`

Reply:
@ishita-gambhir-adobe please correct the typo